documentation/modules/rn-2.5.adoc

```diff
@@ -196,18 +192,11 @@ It is recommended to upgrade from {project-short} 2.4.2 to {project-short} {proj
 
 When upgrading from MTV 2.4.0 to a later version, the operation fails with an error that says the field 'spec.selector' of deployment `forklift-controller` is immutable. Workaround: Remove the custom resource `forklift-controller` of type `ForkliftController` from the installed namespace, and recreate it. Refresh the {ocp} console once the `forklift-console-plugin` pod runs to load the upgraded {project-short} web console. link:https://issues.redhat.com/browse/MTV-518[(MTV-518)]
 
-[NOTE]
-====
-There is an issue with upgrading from 2.4.1 or 2.4.2.
-When upgrading from 2.4 (2.4.z), after the 2.5 operator starts
+[id="deprecated-features-25_{context}"]
+== Deprecated features
 
-. Remove the `mutatingwebhookconfiguration` named `forklift-api` by running the following command:
-+
-`$ oc -n openshift-mtv delete mutatingwebhookconfiguration forklift-api`
+This release has the following deprecated feature:
```

Comment:
@ahadas is this ok to cover - [MTV-727 - deprecated feature](https://issues.redhat.com/browse/MTV-727)

![image](https://github.com/kubev2v/forklift-documentation/assets/106804821/501f3084-61ec-4182-a947-72cd491ea8e6)

Reply:
removed

Reply:
@anarnold97 in #434 you added a screenshot of the modified note but where was it implemented? I don't see it here or in #434 
and we dropped the 'deprecated' part, so do we really need to include it in the release notes?..

Reply:
ah ok, I got it - it's commented out...